.github/workflows/release.yml

```diff
@@ -109,6 +109,25 @@ jobs:
       - name: Publish to galaxy
         run: ansible-galaxy collection publish --api-key=${{ secrets.GALAXY_INFRA_KEY }} ${{ steps.build.outputs.tar_file }}
 
+      - name: Update dependency for Automation Hub
+        run: 'sed -i ''s/fedora\.linux_system_roles/redhat.linux_system_roles/g'' galaxy.yml'
+
+      - name: Update default system roles collection for Automation Hub
+        run: 'sed -i ''s/infra_leapp_upgrade_system_roles_collection: fedora\.linux_system_roles/infra_leapp_upgrade_system_roles_collection: redhat.linux_system_roles/g'' roles/upgrade/defaults/main.yml'
```

Comment:
I'm not sure how to parse this - a double single quote?

Reply:
Yea I think so, but it cleared the error